tools/make-enums.sh

```diff
@@ -2,4 +2,6 @@
 
 set -eux
 
-./tools/make-event-names.py evdev-sys/libevdev/include/linux/input-event-codes.h evdev-sys/libevdev/include/linux/input.hpython tools/make-event-names.py evdev-sys/libevdev/include/linux/input-event-codes.h evdev-sys/libevdev/include/linux/input.h > src/enums.rs
```

Comment:
This line makes absolutely no sense (it looks like a merge conflict where both branches were kept), but worked because the Python script ignored all but the first and last argument.

Reply:
I wonder how it ended up here. Anyway thanks for noticing.